spyder/plugins/breakpoints/plugin.py

```diff
@@ -41,7 +41,7 @@ class Breakpoints(SpyderDockablePlugin):
     Breakpoint list Plugin.
     """
     NAME = 'breakpoints'
-    REQUIRES = [Plugins.Editor]
+    REQUIRES = [Plugins.Editor, Plugins.Debugger]
```

Comment:
@impact27, I'm very sorry but I think I didn't express myself correctly. My idea is not make the breakpoints plugin depend on the debugger one.

Instead, I'd like to remove this plugin, so we don't need an extra dockwidget just to show a table of breakpoints; and to move the functionality present on it to the Debugger plugin.

Reply:
Yes but this is the first step: get the breakpoint logic out of the editor plugin. The merging can happen later

Reply:
Ok, nice! That's great news!